ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/cv_pathfinder_ha.schema.yml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  cv_pathfinder_ha:
```

Comment:
Rethinking about this - let's call it `wan_ha` like the rest to avoid being stuck there in case we end up supporting HA for AutoVPN (which is a future plan)
Sorry for missing this on the first review